modules/rosapps/applications/fraginator/DriveVolume.cpp

```diff
@@ -200,16 +200,28 @@ bool DriveVolume::GetBitmap (void)
     );
 
     // Bad result?
-    if (Result == FALSE  &&  GetLastError () != ERROR_MORE_DATA)
+    if (Result == FALSE && GetLastError() != ERROR_MORE_DATA)
     {
         //wprintf ("\nDeviceIoControl returned false, GetLastError() was not ERROR_MORE_DATA\n");
-        free (Bitmap);
+        HeapFree(GetProcessHeap(), 0, Bitmap);
         return (false);
     }
 
     // Otherwise, we're good
     BitmapSize = sizeof (VOLUME_BITMAP_BUFFER) + (Bitmap->BitmapSize.QuadPart / 8) + 1;
-    Bitmap = (VOLUME_BITMAP_BUFFER *) realloc (Bitmap, BitmapSize);
+
+    void *reallBitmap = HeapReAlloc(GetProcessHeap(), 0, Bitmap, BitmapSize);
+
+    if (reallBitmap == NULL)
+    {
+        //Fail "miserably"
+        //We don't have enought memory left
+        wprintf(L"\nNot enought memory to read volume bitmap\n");
```

Comment:
```suggestion
        // Fail "miserably"
        wprintf(L"\nNot enough memory to read volume bitmap\n");
```
Fix typo in "enought", fix comment style, remove one obvious duplicated line :)